internal/sql/repository/chartConfig/PipelineOverrideRepository.go

```diff
@@ -173,6 +174,16 @@ func (impl PipelineOverrideRepositoryImpl) GetLatestRelease(appId, environmentId
 	return overrides, err
 }
 
+func (impl PipelineOverrideRepositoryImpl) GetLatestReleaseByPipelineIds(pipelineIds []int) (pipelineOverrides []*PipelineOverride, err error) {
+	var overrides []*PipelineOverride
```

Comment:
this query will load all release in the memory or only latest one?

Reply:
> this query will load all release in the memory or only latest one?

all the releases for provided ids.